src/FulfillerSelect.jsx

```diff
@@ -337,15 +339,17 @@ FulfillerSelect.propTypes = {
     includeArchived: PropTypes.bool,
     includeId: PropTypes.bool,
     includeInternalId: PropTypes.bool,
-    includeName: PropTypes.bool
+    includeName: PropTypes.bool,
+    preselectMostRecent: PropTypes.bool
 };
 
 FulfillerSelect.defaultProps = {
     language: 'eng',
     includeArchived: false,
     includeId: true,
     includeInternalId: false,
-    includeName: true
+    includeName: true,
+    preselectMostRecent: true
```

Comment:
It might be good to add a property (or use this one) to not call customizr at all.

Some users might be "confused" by constant re-ordering of the first fulfillers.

Reply:
Thanks, I created #29 for that.